packages/synapse-react-client/stories/FullWidthAlert.stories.tsx

```diff
@@ -53,3 +59,21 @@ export const SuccessAlertWithPrimaryButtonOnly: Story = {
     onClose: undefined,
   },
 }
+
+export const SynapseNavDrawerIsShowing: Story = {
+  args: {
+    ...SuccessAlertWithPrimaryButtonOnly.args,
+  },
+  decorators: [
+    Story => {
+      React.useEffect(() => {
+        document.body.classList.add('SynapseNavDrawerIsShowing')
+        return () => {
+          document.body.classList.remove('SynapseNavDrawerIsShowing')
+        }
+      }, [])
+
+      return <Story />
+    },
+  ],
```

Comment:
Apply the "SynapseNavDrawerIShowing" class to the body to ensure that the story is representative of how the styling will be handled in SWC. Remove the class from the body when leaving the story.

Reply:
This is a great story to add. Not sure if you can override `render` in a story, but if you can, you could even include a `SynapseNavDrawer` component

Reply:
Great idea! Added in the next commit and helped me find an issue with my implementation!